doc/source/news/15.md

```diff
@@ -1,5 +1,18 @@
 # News - 15 series
 
+(release-15-08)=
+## Release 15.08 - 2025-05-08
+
+### Improvements
+
+#### Added support for MySQL 8.0.42 on Ubuntu
+
+This is the latest MySQL release.
+
+#### Added support for MariaDB 10.6.22
```

Comment:
We will add support MariaDB 10.5.29, 10.11.12, and 11.4.6 too.

Reply:
fix: [11ec380](https://github.com/mroonga/mroonga/pull/923/commits/11ec380713908eb0d9f72443e115ce2ae31851ae)